src/pages/Profiel/Profiel.tsx

```diff
@@ -44,7 +65,30 @@ const Profiel = () => {
             "new_email": newEmail
         }
 
-        await back_post_auth("update/email/send/", req, {authState, setAuthState})
+        try {
+            await back_post_auth("update/email/send/", req, {authState, setAuthState})
+            setEditStatus("Verzonden!")
+        } catch (e) {
+
+        }
+    }
+
+    const getRollen = () => {
+        var rollen: string[] = [];
+        authState.scope.split(" ").forEach((item) => {
+            if (item !== "member" && item !== "admin") {
+                if (item === "~2eComCom") {
+                    rollen.push(".ComCom")
+                }
+                else if (item === "NSKEkiden") {
```

Comment:
Deze hardcode mag weg, miss kunnen we sws iets beters bedenken dan dit hardcoden

Reply:
Opzich verandert dit niet veel. Anders moet ik die escape logica ook op de frontend implementeren en dat lijkt me onnodig. NSK Ekiden haal ik wel weg.